interfaces/HO-Portal/src/assets/i18n/en.json

```diff
@@ -275,4 +279,4 @@
     "show": "Show password",
     "hide": "Hide password"
   }
-}
+}
```

Comment:
Something with your local settings/config of Prettier isn't working as it should. :/